saw-remote-api/src/SAWServer/Data/Contract.hs

```diff
@@ -110,7 +111,8 @@ instance FromJSON ty => FromJSON (ContractVar ty) where
 instance (FromJSON ty, FromJSON e) => FromJSON (Contract ty e) where
   parseJSON =
     withObject "contract" $ \o ->
-    Contract <$> o .:  "pre vars"
+    Contract <$> o .:  "mutable globals"
```

Comment:
Mention this new capability of the remote API in the SAW changelog.

Reply:
Ah, good call.